DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -45,11 +45,15 @@ def get_triage_events(self, before_date, after_date):
         response = requests.get(triage_url, headers={"Authorization": "Basic %s" % self.b64val, "searchlight-account-id": "%s" % self.id})
         return response.text
 
-    def get_triage_items(self, item_id_str):
+    def get_triage_items(self, triage_ids):
         """  
             gets triage items from the triage events
         """
-        
-        items_url = self.url + "triage-items?id=" + str(item_id_str)
+
+        item_id_str = triage_ids[0]
+        for ele in triage_ids[1:]:
+            item_id_str = item_id_str + "&id=" + ele
+
+        items_url = self.url + "triage-items?limit=1000&id=" + item_id_str
```

Comment:
added limit because it was limiting to 10 items by default, hence giving incorrect results.